llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/vec-ld.ll

```diff
@@ -483,6 +483,336 @@ define <vscale x 8 x i64>  @vload_nx8i64(ptr %pa) {
   ; RV64-NEXT:   $v8m8 = COPY [[LOAD]](<vscale x 8 x s64>)
   ; RV64-NEXT:   PseudoRET implicit $v8m8
   %va = load <vscale x 8 x i64>, ptr %pa
-	ret <vscale x 8 x i64> %va
+  ret <vscale x 8 x i64> %va
+}
+
+define <vscale x 1 x i8>  @vload_nx1i8_align2(ptr %pa) {
```

Comment:
I am not sure we have the intended coverage on align.

> Align argument could be smaller/equal/larger than element size or smaller/equal/larger than minimum total vector size

For i8 vector types we have test for align 2, align 8, align 32. Recall that align is in bytes but the type i8 is in bits.

Reply:
You are right, I'll fix it!